### PR TITLE
Add tokens for text and bg color

### PR DESCRIPTION
**Pull request by AnnaRybkina**
An attempt to add new tokens for the text and bg colors

1. image
2. image

### CODE REVIEW

**Comment by AnnaRybkina** [comment]
[Engine: add color - should get bg/text from Adi](https://trello.com/c/8JdloUTd/66-engine-add-color-should-get-bg-text-from-adi)

File: src/_rules/color.js [hunk 1]
```diff
@@ -1,3 +1,15 @@
 export const opacity = [
   [/^opacity-?(\d+)$/, ([, d], { theme }) => ({ opacity: theme.opacity[d] }), { autocomplete: 'opacity-${opacity}' }],
 ];
+
+export const textColors = [
+  ['text', { color: 'var(--w-text-color)' }],
+  ['text-inverted', { color: 'var(--w-text-color-inverted)' }, { autocomplete: 'text-inverted' }],
```
Comment (pearofducks) [inline src/_rules/color.js:7]: autocomplete isn't needed for 'static' rules (only ones with regex)

Comment (AnnaRybkina) [reply src/_rules/color.js]: Fixed

**Review by pearofducks — APPROVED** [review]

File: src/_rules/color.js [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+  ['text', { color: 'var(--w-text-color)' }],
+  ['text-inverted', { color: 'var(--w-text-color-inverted)' }],
+  ['text-inverted-subtle', { color: 'var(--w-text-color-inverted-subtle)' }],
+  ['text-subtle', { color: 'var(--w-text-color-subtle)' }],
+];
+
+export const bgColors = [
+  ['bg', { color: 'var(--w-background-color)' }],
+  ['bg-subtle', { color: 'var(--w-background-color-subtle)' }],
+];
```
Comment (nwikstrand) [inline src/_rules/color.js:15]: No new line at the end. Imagine if we had some way of fixing these things before it hits github :trollface:

File: src/_rules/color.js [hunk 3]
```diff
@@ -1,3 +1,15 @@
 export const opacity = [
   [/^opacity-?(\d+)$/, ([, d], { theme }) => ({ opacity: theme.opacity[d] }), { autocomplete: 'opacity-${opacity}' }],
```
Comment (nwikstrand) [inline src/_rules/color.js:2]: Remove optional thingy, code does not cover for anything else

```suggestion
  [/^opacity-(\d+)$/, ([, d], { theme }) => ({ opacity: theme.opacity[d] }), { autocomplete: 'opacity-${opacity}' }],
```

**Review by nwikstrand — COMMENTED** [review]
Not bad at all, good job Anna! 👏  

mind explode

File: src/_rules/color.js (hunk 3 above)
Comment (AnnaRybkina) [reply src/_rules/color.js:2]: Ah true, nice catch

File: src/_rules/color.js [hunk 4]
```diff
@@ -1,3 +1,15 @@
...
+export const textColors = [
+  ['text', { color: 'var(--w-text-color)' }],
+  ['text-inverted', { color: 'var(--w-text-color-inverted)' }],
+  ['text-inverted-subtle', { color: 'var(--w-text-color-inverted-subtle)' }],
+  ['text-subtle', { color: 'var(--w-text-color-subtle)' }],
+];
+
+export const bgColors = [
+  ['bg', { color: 'var(--w-background-color)' }],
+  ['bg-subtle', { color: 'var(--w-background-color-subtle)' }],
```
Comment (BalbinaK) [inline src/_rules/color.js:14]: Shouldn't `bg` be translated to `background-color`? :D

**Pull request merged**